crates/octez/src/async/baker.rs

```diff
@@ -0,0 +1,197 @@
+use crate::{Endpoint, OctezNodeConfig};
+use anyhow::{anyhow, Result};
+use std::{fmt::Display, path::PathBuf};
+use tokio::process::{Child, Command};
+
+use super::{AsyncOctezClient, Protocol};
+
+#[derive(PartialEq, Debug, Clone)]
+pub enum BakerBinaryPath {
+    BuiltIn(Protocol), // The binary exists in $PATH
```

Comment:
Does changing `BuiltIn` to `PATH` cause linting errors?

Reply:
Fair enough, how about `PathVar` or `Pathvar`?

Reply:
renamed it to Env !